### ISSUE TITLE
Class android.text.format.Time has been deprecated since API 22

### ISSUE THREAD

**Issue by LiliWeiSE** [spacecowboy/NotePad#380]
Hi,

I just noticed that class android.text.format.Time is used at many places. Eg, [line 345 in MonthView.java](https://github.com/spacecowboy/NotePad/blob/8b6d082adb164ec637fcc19cf1b021fa5f66d83c/external/datetimepicker/src/com/android/datetimepicker/date/MonthView.java#L345), [line 89 in MonthAdapter](url), etc.

This class has been deprecated for multiple problems (see: https://developer.android.com/reference/android/text/format/Time.html)

Although it may not induce problems for now, I think it'd better to use other alternative classes to avoid future bugs.

Thanks!

**Comment by licaon-kter** [comment]
Yeah, so many issue, great app, too bad it's not developed anymore.

Check out Orgzly, they already merged reminders o I guess I'll jump ship. BTW, you can import these notes in Orgzly, albeit with a caveat that I forget now.

**Comment by CampelloManuel** [comment]
Google recommends using `GregorianCalendar` https://developer.android.com/reference/java/util/GregorianCalendar instead of `android.text.format.Time` which we use. A limit of that class is that it can't represent dates after 2038. So replacing `Time` is important, even more than other deprecated classes used in the codebase
